blockchains/elrond/assets/USDC-c76f1f/info.json

```
@@ -0,0 +1,28 @@
+{
+    "type": "ESDT",
+    "id": "USDC-c76f1f",
+    "name": "WrappedUSDC",
+    "symbol": "USDC",
+    "decimals": 6,
+    "status": "active",
+    "description": "USDC stablecoin originating on Ethereum, bridged as an ESDT token on Elrond. 1 USDC = 1 WrappedUSDC",
```

```suggestion
    "description": "USDC stablecoin originating on Ethereum, bridged as an ESDT token on Elrond. 1 USDC = 1 Wrapped USDC",
```

Same for description

---

Fixed.